pkg/ioutils/prompt.go

```diff
@@ -6,10 +6,19 @@ Not licensed for re-use.
 
 package ioutils
 
-import "github.com/manifoldco/promptui"
+import (
+	"github.com/manifoldco/promptui"
+	"github.com/mattn/go-isatty"
+	"os"
+)
 
 // PromptRunner is the interface that wraps the promptui.Prompt and makes a call
 // to it from the aspect CLI Core.
 type PromptRunner interface {
 	Run(prompt promptui.Prompt) (string, error)
 }
+
+// Check if the CLI can be run in interactive mode
+func IsInteractive() bool {
```

Comment:
you shouldn't check this, because there's also a `--interactive` flag that can be passed. Check Thulio's recent change for how to observe the value

Reply:
there's existing logic that consults both the flag, and the execution environment, you need to find and reuse that

Reply:
@alexeagle I synced with Chai yesterday over a call to go over this already.